script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -127,23 +127,23 @@ Even if it leaks, it'll be nothing more than a rumor. Besides, there's nothing i
 [sha:68e29f8b315c8e3e2ce21da1695e602aecf16ecb]{
 -- Page 796, Offset 13010. Glued. Mods: @x.
 -- 坊ちゃん、その歳でエリートコースまっしぐらッスね。家柄的な話プラス、精神的な話で。なんかこう、もっと年相応の遊び心とかないんですか？」
- Young master, at your age, you're going on the fast track at full speed. Speaking of your pedigree and mentality. I mean, you don't feel the desire to act a bit more like your age and have some fun?"
+ Young master, at your age, you're going on the fast track at full speed. Speaking of your pedigree and mentality. I mean, don't you feel the desire to act a bit more like your age and have some fun?"
 }
 [sha:6b5a46592fca4924d9eda81d955c113f1bc5c10d]{
 -- Page 797, Offset 13011.
 -- 「あるかンなもん。ガキってのは基本つまらねえもんだって忘れたのか30歳？　オラ、無駄話してねえで監督してきやがれ。ヤバめな証拠は隠滅しとけよ」
 // He says 30歳, thirty year-old. Just imagine that I thought about "30 year old boomer" for a second haha
-"No way, I don't. You've forgotten that brats are just basically a pain, you geezer? Hey, stop talking nonsense and go supervise. And make sure to destroy any dangerous evidence."
+"No way, I don't. Have you forgotten that brats are pretty much all dull, you 30 year old geezer? Hey, stop talking nonsense and go supervise. And make sure to destroy any dangerous evidence."
```

Comment:
No fucking way. Have you forgotten that brats are all dull, you 30-year-old-geezer? Quit it with your nonsense and go supervise. And make sure to destroy any dangerous evidence.